### PR TITLE
1190: Increase comprehensibility of application form

### PR DESCRIPTION
**Pull request by ztefanie**
### Short description

Changed wording of heading and changed order as suggested in the ticket

### Resolved issues

Fixes: #1190

### CODE REVIEW

File: administration/src/mui-modules/application/forms/WorkAtOrganizationForm.tsx [hunk 1]
```diff
@@ -67,11 +67,7 @@ const WorkAtOrganizationForm: Form<State, Options, ValidatedInput, AdditionalPro
   Component: ({ state, setState, onDelete }) => (
     <>
       <ActivityDivider onDelete={onDelete} />
-      <SubForms.organization.Component
-        state={state.organization}
-        setState={useUpdateStateCallback(setState, 'organization')}
-      />
-      <h4>Angaben zur Tätigkeit</h4>
+      <h4>Angaben zu ihrer ehrenamtlichen Tätigkeit</h4>
```
Comment (michael-markl) [inline administration/src/mui-modules/application/forms/WorkAtOrganizationForm.tsx:70]:
```suggestion
      <h4>Angaben zu Ihrer ehrenamtlichen Tätigkeit</h4>
```

**Review by sarahsporck — COMMENTED** [review]
I think you missed the second list point "Angaben zur Organisation und Ihrem Ehrenamt".

**Comment by ztefanie** [comment]
> I think you missed the second list point "Angaben zur Organisation und Ihrem Ehrenamt".

This was on purpose. I think this renaming does not really make sense, as in the section this heading belongs to, you only put information about the organization. Of course I can rename it, if you think this should be done.

**Comment by sarahsporck** [comment]
> > I think you missed the second list point "Angaben zur Organisation und Ihrem Ehrenamt".
> 
> This was on purpose. I think this renaming does not really make sense, as in the section this heading belongs to, you only put information about the organization. Of course I can rename it, if you think this should be done.

Fine with me, as long, as you can justify why you left it :).

**Review by sarahsporck — APPROVED** [review]
Tested in Firefox

**Review by f1sh1918 — APPROVED** [review]
Tested on chrome

**Pull request merged**